docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
   activemq:
     build:
       context: ./activemq
-    image: oapass/activemq:20180614@sha256:2afadbccda09448ec67eb6d830314d2b690b83f1c535d0fbbefaee7ec358bfcc
+    image: oapass/activemq:20180614
```

Comment:
Is it OK to leave this un-pinned?  Or has it not been pushed?

Reply:
@emetsger ugh, good catch.  let me push it and pin it to the hash